### DISCUSSION TITLE
(bug or not?) Restful API couldn't get the docker image correctly for an org

### DISCUSSION THREAD

**Discussion by wqhhust** [community/community#53854] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

When I use the following API to get all docker images, it returns an empty list
 https://api.github.com/orgs/ORG/packages?package_type=container
![image](https://user-images.githubusercontent.com/5955201/234802457-ef2f5102-d424-46b0-8b37-6ff64e2dd249.png)

But actually it has many docker images. 
![image](https://user-images.githubusercontent.com/5955201/234803107-c751b8c9-6c29-4b7c-8d99-a419070d068e.png)

I am not sure whether this is a bug or not

**Comment by wqhhust** [comment]
The url I am using is https://api.github.com/orgs/FerretDB/packages?package_type=docker

**Comment by airtower-luna** [comment]
You need to use `package_type=container`, not `docker`. The [`package_type` description mentions](https://docs.github.com/en/rest/packages?apiVersion=2022-11-28#list-packages-for-an-organization) that `container` will return container images on `ghcr.io`, while `docker` is for the deprecated `docker.pkg.github.com`.

**Reply by wqhhust** [reply]
Thanks very much for your reply. 
I have tried to use "container", but doesn't return the images that I need 
![image](https://user-images.githubusercontent.com/5955201/234901845-9355433c-ccc5-43ef-8a71-c492be13c461.png)

What I need are the tagged images versions like below:
![image](https://user-images.githubusercontent.com/5955201/234902473-abe5218a-31c7-4b6f-ad85-b275d9d08217.png)

**Reply by airtower-luna** [reply]
In that case you're looking at the wrong API endpoint, the one you used just lists the packages (containers, in your case) the organization has. If you want to get the versions available for a specific package, you need to use this endpoint: https://docs.github.com/en/rest/packages?apiVersion=2022-11-28#list-package-versions-for-a-package-owned-by-an-organization

So for your image, the URL will be something like this:

```
https://api.github.com/orgs/FerretDB/packages/container/ferretdb-dev/versions
```

If you want to know the available tags for all the packages, you'll need to combine the two: First get the list of packages, then iterate over them and get the versions for each.
